src/test/java/uk/gov/dhsc/htbhf/local/LocalConfiguration.java

```diff
@@ -43,18 +46,27 @@ public WebDriverWrapper localWebDriverWrapper() {
         return new LocalWebDriverWrapper(browser, headless, waitTimeoutInSeconds, baseUrl);
     }
 
-    @Bean(destroyMethod = "stop")
-    public WireMockServer wireMockServer() {
-        WireMockServer wireMockServer = new WireMockServer(wireMockConfig().port(wiremockPort));
+    @Bean(name="claimantServiceMock", destroyMethod = "stop")
+    public WireMockServer claimantServiceMock() {
+        WireMockServer wireMockServer = new WireMockServer(wireMockConfig().port(claimantServicePort));
+        wireMockServer.start();
+        //Configure the WireMock client to use the same port configured for the server.
+        WireMock.configureFor(claimantServicePort);
+        return wireMockServer;
+    }
+
+    @Bean(name="osPlacesMock", destroyMethod = "stop")
+    public WireMockServer osPlacesMock() {
+        WireMockServer wireMockServer = new WireMockServer(wireMockConfig().port(osPlacesPort));
```

Comment:
These two bean creation methods are identical apart from the port number, can we refactor out the common code please?